diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/utils/RulesConfigYamlTest.kt

```diff
@@ -19,12 +19,13 @@ import kotlinx.serialization.encodeToString
  * Special test that checks that developer has not forgotten to add his warning to a diktat-analysis.yml
  * This file is needed to be in tact with latest changes in Warnings.kt
  */
-@JvmInline
 @Suppress("UNUSED")
-value class RulesConfigYamlTest(private val pathMap: Map<String, String> =
-        mapOf("diktat-analysis.yml" to "diKTat/diktat-rules/src/main/resources/diktat-analysis.yml",
-            "diktat-analysis-huawei.yml" to "diKTat/diktat-rules/src/main/resources/diktat-analysis-huawei.yml",
-            "parent/diktat-analysis.yml" to "diKTat/diktat-analysis.yml")) {
+class RulesConfigYamlTest {
+    private val pathMap: Map<String, String> =
+            mapOf("diktat-analysis.yml" to "diKTat/diktat-rules/src/main/resources/diktat-analysis.yml",
+                "diktat-analysis-huawei.yml" to "diKTat/diktat-rules/src/main/resources/diktat-analysis-huawei.yml",
+                "../diktat-analysis.yml" to "../diktat-analysis.yml")
```

Comment:
I don't think these values should be changed, we construct the path relative to maven's `user.dir` for the third config anyway.